aether-kernel/aether/kernel/api/serializers.py

```diff
@@ -203,7 +203,7 @@ class AttachmentSerializer(DynamicFieldsMixin, serializers.ModelSerializer):
 
     submission = MtPrimaryKeyRelatedField(
         queryset=models.Submission.objects.all(),
-        mt_field='mappingset__project',
+        mt_field='project',
```

Comment:
Also int the `class SubmissionSerializer` the `mappingset` field is no longer required ;-)

Reply:
But I would say that **IT IS REQUIRED TO CREATE THE SUBMISSION** or no entities will be extracted...